### ISSUE TITLE
How to Install the SCIP-SDP Plugin on Windows?

### ISSUE THREAD

**Issue by Amazing-Birds** [scipopt/SCIP-SDP#18]
I am a beginner in MISDP and would like to further study this problem. However, I encountered some difficulties while installing the plugin and would appreciate your guidance. Thank you for your valuable time.

Could you please advise on the specific differences or additional considerations that need to be taken into account when installing on a Windows system, as opposed to Linux? I noticed that the installation file is based on Linux instructions, but I encounter various parameter issues when compiling with MinGW on Windows. Could these problems be caused by system differences?

**Comment by pfetsch** [comment]
Yes, the installation is mainly written for Linux, but should be very similar on MinGW (or the WSL). Have you been able to compile SCIP?

**Comment by Amazing-Birds** [comment]
Since I directly downloaded the SCIP for Windows, no additional compilation was needed during installation. In addition, I would like to inquire further: given that I am currently using Python to construct the MISDP model, after installing the SCIP-SDP plugin, will I be able to use SCIP-SDP through Python?

**Comment by pfetsch** [comment]
So far, there is no Python interface to SCIP-SDP. Depending on the interest of users, I might implement one in the future.

**Comment by FranekStark** [comment]
@pfetsch what is the status on that?  I would be interested in a python interface.

**Comment by pfetsch** [comment]
> @pfetsch what is the status on that? I would be interested in a python interface.

Thanks for your interest. I will put it on my list, but this might take a while.

**Comment by mn22nazerian** [comment]
@pfetsch Is there a method to install SCIP-SDP on windows?

**Comment by pfetsch** [comment]
> [@pfetsch](https://github.com/pfetsch) Is there a method to install SCIP-SDP on windows?

No specific method. You should follow the installation process of SCIP, since this needs to be compiled by you anyway. The easiest way is probably to use the WSL.
